napytau/ingest/reader/file_reader.py

```diff
@@ -0,0 +1,21 @@
+from os.path import isfile
+from typing import List
+
+from napytau.ingest.reader.reader import Reader
+
```

Comment:
It may be a bit confusing to have a class called "reader" inside a directory called "reader"